### PR TITLE
add in predictive back snippets for NavHost and predictive back handler

### PR DESCRIPTION
**Pull request by trambui09**
Add in the predictive back snippets for NavHost implementation and the Predictive Back Handler for a navigation drawer

### CODE REVIEW

File: compose/snippets/src/main/java/com/example/compose/snippets/predictiveback/PredictiveBackSnippets.kt [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.input.pointer.util.VelocityTracker
+import androidx.compose.ui.platform.LocalDensity
+import kotlin.coroutines.cancellation.CancellationException
+import kotlinx.coroutines.launch
+
+
+@Composable
+fun MainNavigation(
```
Comment (riggaroo) [inline compose/snippets/src/main/java/com/example/compose/snippets/predictiveback/PredictiveBackSnippets.kt]: You can make these private and possible name it PredictiveBackOverrideExit?

File: compose/snippets/src/main/java/com/example/compose/snippets/predictiveback/PredictiveBackSnippets.kt [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+@Composable
+fun SettingsScreen(
+    modifier: Modifier = Modifier, navController: NavHostController
+) {
+
+}
+
+@Composable
+fun HomeScreenDrawer() {
```
Comment (riggaroo) [inline compose/snippets/src/main/java/com/example/compose/snippets/predictiveback/PredictiveBackSnippets.kt]: same thing here, you can make it private and more descriptive about what the snippet shows.

File: compose/snippets/src/main/java/com/example/compose/snippets/predictiveback/PredictiveBackSnippets.kt (hunk 1 above)
Comment (trambui09) [reply compose/snippets/src/main/java/com/example/compose/snippets/predictiveback/PredictiveBackSnippets.kt]: Yeah!

**Comment by trambui09** [comment]
Closing PR as this is a dup of a merged PR.

**Pull request closed without merging**